commands.json

```diff
@@ -30,5 +282,878 @@
+    "arguments": [
+      {
+        "name": "dict",
+        "type": "key"
+      }
+    ],
+    "since": "1.0.0",
+    "group": "search"
+  },
+  "FT._LIST": {
+    "summary": "Returns a list of all existing indexes",
+    "complexity": "O(1)",
+    "since": "2.0.0",
+    "group": "search"
+  },
+  "FT.CONFIG SET": {
+    "summary": "Set runtime configuration options",
+    "complexity": "O(1)",
+    "arguments": [
+      {
+        "name": "option",
+        "type": "string"
+      }
+    ],
+    "since": "1.0.0",
+    "group": "search"
+  },
+  "FT.CONFIG GET": {
+    "summary": "Retrieve runtime configuration options",
+    "complexity": "O(1)",
+    "arguments": [
+      {
+        "name": "option",
+        "type": "string"
+      },
+      {
+        "name": "value",
+        "type": "string"
+      }
+    ],
+    "since": "1.0.0",
+    "group": "search"
+  },
+  "FT.CONFIG HELP": {
+    "summary": "Help description of runtime configuration options",
+    "complexity": "O(1)",
+    "arguments": [
+      {
+        "name": "option",
+        "type": "string"
+      }
+    ],
+    "since": "1.0.0",
+    "group": "search"
+  },
+  "FT.SEARCH": {
+    "summary": "Searches the index with a textual query, returning either documents or just ids",
+    "complexity": "O(N)",
+    "arguments": [
+      {
+        "name": "index",
+        "type": "key"
+      },
+      {
+        "name": "query",
+        "type": "string"
+      },
+      {
+        "name": "nocontent",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "NOCONTENT"
+        ]
+      },
+      {
+        "name": "verbatim",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "VERBATIM"
+        ]
+      },
+      {
+        "name": "nostopwords",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "NOSTOPWORDS"
+        ]
+      },
+      {
+        "name": "withscores",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "WITHSCORES"
+        ]
+      },
+      {
+        "name": "withpatloads",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "WITHPAYLOADS"
+        ]
+      },
+      {
+        "name": "withsortkeys",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "WITHSORTKEYS"
+        ]
+      },
+      {
+        "name": "withpatloads",
+        "type": "enum",
+        "optional": true,
+        "enum": [
+          "WITHPAYLOADS"
+        ]
+      },
+      {
+        "name": "filter",
+        "type": "block",
+        "optional": true,
+        "multiple": true,
+        "block": [
+          {
+            "name": "numeric_field",
+            "type": "string",
+            "command":"FILTER"
+          },
+          {
+            "name": "min",
+            "type": "double"
+          },
+          {
+            "name": "max",
+            "type": "double"
```

Comment:
Need to also mention `+inf` (use a more complex enum?)

Reply:
Yes.
Due to the current limitations, lets keep it as is.